cd/manager/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ekidd/rust-musl-builder:stable as builder
+
+RUN USER=root cargo new --bin ceramic-cd-manager
+WORKDIR ./ceramic-cd-manager
+COPY ./Cargo.lock ./Cargo.lock
+COPY ./Cargo.toml ./Cargo.toml
+RUN cargo build --release
```

Comment:
Compiling in the docker image? This means we can't cache this step correct?

Reply:
We can actually cache this step.